easybuild/tools/job/pbs_python.py

```diff
@@ -115,23 +170,17 @@ def __init__(self, script, name, env_vars=None, resources={}, conn=None, ppn=Non
             self.env_vars = {}
         self.name = name
 
-        if pbs_import_failed:
-            self.log.error(pbs_import_failed)
-
         try:
-            self.pbs_server = pbs.pbs_default()
-            if conn:
-                self.pbsconn = conn
-                self.clean_conn = False
-            else:
-                self.pbsconn = pbs.pbs_connect(self.pbs_server)
+            self.pbsconn = self._server.connect_to_server()
         except Exception, err:
-            self.log.error("Failed to connect to the default pbs server: %s" % err)
+            self.log.error("Failed to connect to PBS server: %s" % err)
+            raise
```

Comment:
It's harmless: if `log.error()` raises itself an exception, line 177 is never reached.  Else, it's a safeguard: when "raise on error" is off, the exception is still propagated instead of (unwttingly) ignored.